viofs/pci/ioctl.c

```diff
@@ -32,9 +32,9 @@
 
 EVT_WDF_REQUEST_CANCEL VirtFsEvtRequestCancel;
 
-static int GetVirtQueueIndex(IN PDEVICE_CONTEXT Context)
+static int GetVirtQueueIndex(IN PDEVICE_CONTEXT Context, IN BOOLEAN HighPrio)
```

Comment:
This function is small and used on a data path. Worth thinking to change it to be "inline".